src/researchhub_case/migrations/0008_authorclaimcase_claim_context.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2 on 2022-02-11 20:24
```

Comment:
A few migrations but the net change is two new fields to author claim table:
`context_content_type_id` and `context_content_id`